### ISSUE TITLE
Rating star count on mouseover

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#418]
```
It's difficult to see, at a glance, how many stars are being selected.  I find 
myself going to 10 and counting back one star at a time down to 6 or 7 as 
needed.

Would make it easier to show the star count on mouseover when the user is going 
to rate a show.  Or display the stars with the number in the middle.
```

Original issue reported on code.google.com by `Ivan...@gmail.com` on 15 Mar 2013 at 10:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `werndly...@gmail.com` on 6 May 2013 at 12:05
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Product-JMM**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.32
```

Original comment by `werndly...@gmail.com` on 9 May 2013 at 1:19
- Changed state: **Fixed**
